plover/gui/config.py

```diff
@@ -46,6 +46,7 @@
 SPACE_PLACEMENT_BEFORE = "Before Output"
 SPACE_PLACEMENT_AFTER = "After Output"
 SPACE_PLACEMENTS = [SPACE_PLACEMENT_BEFORE, SPACE_PLACEMENT_AFTER]
+TRANSLATION_TRANSPARENCY_LABEL = "Translation Dialog Transparency"
```

Comment:
The awkward bit is that ultimately we pass this into wx SetTransparency. Which is actually SetOpacity. But I think we could add a comment at that call site acknowledging the confusion and be clearer through the rest of the code by adopting the opacity verbiage.

--
Jeremy W. Sherman
http://jeremywsherman.com/

> El 20-02-2016, a las 18:17, Ted Morin <notifications@github.com> escribió:
> 
> In plover/gui/config.py:
> 
> > @@ -46,6 +46,7 @@
> >  SPACE_PLACEMENT_BEFORE = "Before Output"
> >  SPACE_PLACEMENT_AFTER = "After Output"
> >  SPACE_PLACEMENTS = [SPACE_PLACEMENT_BEFORE, SPACE_PLACEMENT_AFTER]
> > +TRANSLATION_TRANSPARENCY_LABEL = "Translation Dialog Transparency"
> Translation Dialog Opacity, now that we've flipped the slider?
> 
> —
> Reply to this email directly or view it on GitHub.
>